### DISCUSSION TITLE
Example code error

### DISCUSSION THREAD

**Discussion by De-Jeriyo** [makerspet/support#12] [General]
I run this example code 
"ldrobot_ld14p_esp32.ino" 
and got this error 
"c:\Users\User\Documents\Arduino\libraries\LDS\src\LDS.cpp: In member function 'virtual String LDS::resultCodeToString(result_t)':
c:\Users\User\Documents\Arduino\libraries\LDS\src\LDS.cpp:123:10: error: 'OK' was not declared in this scope
  123 |     case OK:
      |          ^~

Using library LDS at version 0.5.4 in folder: C:\Users\User\Documents\Arduino\libraries\LDS 
exit status 1

Compilation error: exit status 1"

**Comment by kaiaai** [comment]
I'm checking in a fix. Please delete your LDS library, download it again and retry compiling.
